2/README.md

```diff
@@ -34,25 +34,26 @@ Specifications should take minutes to explain, hours to design, days to write, w
-* Each specification is a set of wiki pages, together with comments, attached files, and other resources.
-* Important specifications may also exist as subdomains, i.e. child wikis.
+The author of the specification is responsible for building consensus within the community and documenting dissenting opinions.
 
-Individuals can become members of the domain by completing the necessary legal clearance. The copyright, patent, and trademark policies of the domain must be clarified in an Intellectual Property policy that applies to the domain.
-
-Specifications exist as multiple pages, one page per version of the specification (see "Branching and Merging", below), which may be assigned URIs that include an incremental number. Thus, we refer to a specification by specifying its domain, number, and short name. New versions of the same specification will have new numbers. The syntax for a specification reference is:
+## Architecture
 
+### Types of specification
+There are three types of specifications:
+* A **Standard Track Specification** describes any change to network protocols, transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using BigchainDB products.
+* An **Informational Specification** describes a BigchainDB design issue, or provides general guidelines or information to the BigchainDB community, but does not propose a new feature. Informational Specifications do not necessarily represent BigchainDB community consensus or a recommendation, so users and implementers are free to ignore Informational Specifications or follow their advice.
+* A **Meta Specification** describes a process surrounding BigchainDB or proposes a change to a process.
 
-  <domain>/spec:<number>/<shortname>
+### Specification Format
+A specification is a set of Markdown documents (the main file SHOULD be called `README.md`), together with comments, attached files, and other resources. A specification is identified by its number and short name (e.g. this specification is **2/COSS**). The number of the specification is also the name of the directory where its files are stored.
 
+Every specification (including branches) carries a different number. New versions of the same specification have new numbers.
 
-For example, this specification is **rfc.unprotocols.org/spec:2/COSS**. The short form **2/COSS** may be used when referring to the specification from other specifications in the same domain.
+### Specification template
+TBD: Embed Muawia work here https://github.com/bigchaindb/bigchaindb/tree/master/proposals
 
-Every specification (including branches) carries a different number. Lower numbers indicate more mature specifications, higher numbers indicate more experimental specifications.
+### Copyright Waiver
+All Specifications MUST be in public domain. The only two exceptions are 1/C4 and 2/COSS, licensed under GPL.
```

Comment:
How about, "Except for 1/C4 and 2/COSS, all Specifications MUST be released to the public domain."

(Anyone curious about how 1/C4 and 2/COSS are licensed can look in the **License** section of those documents.)

Maybe we could provide an example sentence that they could use to release the spec to the public domain. Here's one I got by using the [Creative Commons CC0 wizard](https://creativecommons.org/choose/zero/):

To the extent possible under law, the Editor and Contributors listed above have waived all copyright and related or neighboring rights to TITLE OF THIS DOCUMENT. This work is published from: INSERT COUNTRY HERE.

Reply:
Thanks @ttmc, I didn't know about the CC0 wizard. If you do it without filling the form, you get:
>  To the extent possible under law, the person who associated CC0 with this work has waived all copyright and related or neighboring rights to this work.